tools/ci_build/github/linux/run_build.sh

```diff
@@ -53,7 +53,16 @@ else
             --use_cuda \
             --cuda_home /usr/local/cuda \
             --cudnn_home /usr/local/cudnn-$_CUDNN_VERSION/cuda $BUILD_EXTR_PAR
-    elif [ $BUILD_DEVICE = "tensorrt" ]; then
+    elif [[ $BUILD_DEVICE = "tensorrt"* ]]; then
+        if [ $BUILD_DEVICE = "tensorrt-v7.1" ]; then
+            CUR_PWD=$(pwd)
+            cd $SCRIPT_DIR/../../../../
+            chmod 775 .gitmodules
+            git config --file=.gitmodules submodule.cmake/external/onnx-tensorrt.branch 7.1
+            git submodule update --init --remote cmake/external/onnx-tensorrt
```

Comment:
I think you don't need to update gitmodules. You can directly go to the "cmake/external/onnx-tensorrt" folder and run
```
git remote update
git checkout 7.1
```

Reply:
Yes, I've changed the script.